eventuate-local-java-cdc-connector-common/src/main/java/io/eventuate/local/common/MessageCleaner.java

```diff
@@ -0,0 +1,94 @@
+package io.eventuate.local.common;
+
+import io.eventuate.common.jdbc.EventuateSchema;
+import io.eventuate.common.jdbc.sqldialect.EventuateSqlDialect;
+import org.springframework.jdbc.core.JdbcTemplate;
+
+import javax.sql.DataSource;
+import java.util.Timer;
+import java.util.TimerTask;
+
+public class MessageCleaner {
+
+  private EventuateSqlDialect eventuateSqlDialect;
+  private DataSource dataSource;
+  private EventuateSchema eventuateSchema;
+  private Boolean purgeMessagesEnabled;
+  private int purgeMessagesMaxAgeInSeconds;
+  private Boolean purgeReceivedMessagesEnabled;
+  private int purgeReceivedMessagesMaxAgeInSeconds;
+
+  private int purgeIntervalInSeconds;
+
+  private Timer timer;
+  private JdbcTemplate jdbcTemplate;
+
+
+  public MessageCleaner(EventuateSqlDialect eventuateSqlDialect,
+                        DataSource dataSource,
+                        EventuateSchema eventuateSchema,
+                        Boolean purgeMessagesEnabled,
```

Comment:
This pattern of fields is repeated:

```
Boolean purgeMessagesEnabled,
                        int purgeMessagesMaxAgeInSeconds,
                        Boolean purgeReceivedMessagesEnabled,
                        int purgeReceivedMessagesMaxAgeInSeconds,
                        int purgeIntervalInSeconds
```

There should be a class CleanerConfiguration? that is passed around.

Reply:
ok, I will add